Buildings/Fluid/SolarCollectors/BaseClasses/ASHRAESolarGain.mo

```diff
@@ -49,6 +49,10 @@ block ASHRAESolarGain
     annotation (Placement(transformation(extent={{100,-10},{120,10}})));
 
 protected
+  constant Modelica.SIunits.Temperature dTMax = 1
```

Comment:
We might as well make this an advanced parameter? I recall having problems before with the - 1 K not being sufficient although right now I do not understand how this could pose a problem.